packages/es-components/src/components/base/icons/IconContext.ts

```diff
@@ -13,7 +13,16 @@ class NodeError extends Error {
 }
 
 const iconStyleAttribute = 'data-es-icon-styles';
-const iconsAsset = `${ASSETS_PATH}icons.css`;
+let iconsAsset = `${ASSETS_PATH}icons.css`;
+
+const subdomain = window.location.hostname.split(".")[1];
+const environments:string[] = ['dev','qa','stage'];
+const local: string = 'localtest';
```

Comment:
You don't need specific typing for this. The compiler can infer types when you directly give variables values.
```suggestion
const environments = ['dev','qa','stage'] as const;
const local = 'localtest';
```